apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -555,3 +620,12 @@ NetSimPacketEditor.prototype.updateBitCounter = function () {
 NetSimPacketEditor.prototype.onRemovePacketButtonClick_ = function () {
   this.removePacketCallback_(this);
 };
+
+/**
+ * Remove the first bit of the packet binary, used when sending one bit
+ * at a time.
+ */
+NetSimPacketEditor.prototype.consumeFirstBit = function () {
+  this.setPacketBinary(this.getPacketBinary().substr(1));
```

Comment:
Ok, I misread this the first time through, thinking they had the same substring logic. Looks good :)